roles/usegalaxy-eu.fix-stuck-handlers/defaults/main.yml

```diff
@@ -13,11 +13,10 @@ cron_tasks:
   - name: "Restart gunicorn"
     minute: 30
     hour: 3
-    job: "systemctl restart galaxy-gunicorn"
+    job: "/bin/bash -c 'for (( c=0; c<{{ galaxy_systemd_gunicorns }}; c++ )); do systemctl restart galaxy-gunicorn@$c && sleep {{ sleep_duration }}; done'"
```

Comment:
Are 4 minutes of sleep duration enough?

Reply:
I can add a minute or two

Reply:
Yeah, that would be better I think. Better safe than oops!

Reply:
Shouldn't we (not now, but later) scan the first gunicorn if its up ... scanning for the port and or socket that exists? Or something in the logs that indicate its up? I think Nate has something in gravity for that?